samples/simple-fsharp-console/FSharpConsole/Program.cs

```diff
@@ -0,0 +1,58 @@
+﻿using Microsoft.DotNet.Interactive;
+using Microsoft.DotNet.Interactive.Commands;
+using Microsoft.DotNet.Interactive.Events;
+using Microsoft.DotNet.Interactive.Formatting;
+using Microsoft.DotNet.Interactive.FSharp;
+using System;
+using System.Linq;
+
+var kernel = new FSharpKernel()
+                    .UseDefaultFormatting()
+                    .UseNugetDirective()
+                    .UseKernelHelpers()
+                    .UseWho()
+                    .UseDotNetVariableSharing();
+Formatter.SetPreferredMimeTypeFor(typeof(object), "text/plain");
+Formatter.Register<object>(o => o.ToString());
+
+while (true)
+{
+    if (ReadLine() is not { } request)
+        continue;
+    var toSubmit = new SubmitCode(request);
+    var response = kernel.SendAsync(toSubmit).Result;
```

Comment:
It's always preferable to `await` and avoid using `.Result`.

Reply:
The point was to show a synchronous sample without any complexity at all. But if it's required, I will change it too.

Reply:
The `await` will still freeze the interface, and it's two fewer non-whitespace characters. 😄 

I have a general no-`.Result` rule because of the deadlock potential. Using `await` is a good habit to instill.